test/AutoDiff/builtin_math.sil

```diff
@@ -1,3 +1,4 @@
+// RUN: %target-swift-frontend -emit-sil -O %s

```

Comment:
Is this a typo? You're running the test twice with identical parameters, and only file checking the second run